europace_security.yaml

```
@@ -70,6 +70,7 @@ flows:
         ## Kreditsmartangebote ermitteln
       privatkredit:antrag:schreiben: |
         ## Kreditsmartanträge schreiben
+        Der Client kann den Antragsstatus verändern oder den aktuellen Status um Zusatzinformationen ergänzen.
```

Vielleicht eher sowas oder ist da zuviel Dopplung?
```suggestion
        Der Client kann den Antragsstatus aktualisieren oder den aktuellen Status um Zusatzinformationen ergänzen.
```

---

Kein Einwand. Meine Version habe ich von https://docs.api.europace.de/privatkredit/produktanbieter/kex-antragsstatus-api/ kopiert. Vielleicht dort auch ändern?

---

Guter Punkt, dann halten wir es erstmal konsistent und lassen es bei der 1. Variante.